kpet/utils.py

```diff
@@ -19,17 +19,17 @@ class TemplateNotFound(Exception):
     """Raised when an template is not found"""
 
 
-def get_template_path(filename):
+def get_template_path(filename, dbdir=None):
     """Return the full path for the corresponding template"""
-    current_dir = os.path.realpath(os.path.dirname(__file__))
-    path = os.path.join(current_dir, 'templates', filename)
+    workdir = dbdir or os.path.realpath(os.path.dirname(__file__))
+    path = os.path.join(workdir, 'templates', filename)
```

Comment:
Since we're making the `--db` option required now, do you think you could just make the `dbdir` a required argument?

Reply:
Let's do that later in other PR.